docs/mast/mast.rst

```diff
@@ -181,6 +180,111 @@ To get a table of metadata associated with observation or product lists use the
+                    PUBLIC NUCLEUS+HODGE602         OBQU010F0 0.022143836477276503      12556 210.80227 54.34895           GORDON 210.80227 54.34895OBQU010F0
+                    PUBLIC NUCLEUS+HODGE602         OBQU010J0  0.04381046755938432      12556 210.80227 54.34895           GORDON 210.80227 54.34895OBQU010J0
+
+
+for paging through the results, offset and limit can be used to specify the starting record and the number
+of returned records. the default values for offset and limit is 0 and 5000 respectively.
+
+.. code-block:: python
+
+                >>> from astroquery.mast.missions import MastMissions
+                >>> from astropy.coordinates import SkyCoord
+                >>> missions = MastMissions()
+                >>> results = missions.query_criteria(sci_start_time=">=2021-01-01 00:00:00", select_cols=["sci_stop_time", "sci_targname", "sci_start_time", "sci_status", "sci_pep_id"], sort_by=['sci_pep_id'], limit=1000, offset=1000)
+                >>> len(results)
+                >>> 1000 
+
+Meta data search can also be performed using object names like M101.
+
+.. code-block:: python
+
+                >>> results = missions.query_object('M101', 3, select_cols=["sci_stop_time", "sci_targname", "sci_start_time", "sci_status"], sort_by=['sci_targname'])
+                >>> results[:5]
+                <Table masked=True length=5>
+                ang_sep           search_pos     sci_status          search_key               sci_stop_time        sci_targname       sci_start_time       sci_data_set_name
+                str20              str18           str6               str27                      str26               str16               str26                   str9      
+                ------------------ ------------------ ---------- --------------------------- -------------------------- ------------ -------------------------- -----------------
+                2.751140575012458  210.80227 54.34895     PUBLIC 210.80227 54.34895LDJI01010 2019-02-19T05:52:40.020000   +164.6+9.9 2019-02-19T00:49:58.010000         LDJI01010
+                0.8000626246647815 210.80227 54.34895     PUBLIC 210.80227 54.34895J8OB02011 2003-08-27T08:27:34.513000   ANY        2003-08-27T07:44:47.417000         J8OB02011
+                1.1261718338567348 210.80227 54.34895     PUBLIC 210.80227 54.34895J8D711J1Q 2003-01-17T00:50:22.250000   ANY        2003-01-17T00:42:06.993000         J8D711J1Q
+                1.1454431087675097 210.80227 54.34895     PUBLIC 210.80227 54.34895JD6V01012 2017-06-15T18:33:25.983000   ANY        2017-06-15T18:10:12.037000         JD6V01012
+                1.1457795862361977 210.80227 54.34895     PUBLIC 210.80227 54.34895JD6V01013 2017-06-15T20:08:44.063000   ANY        2017-06-15T19:45:30.023000         JD6V01013
+
+Meta search can be performed by using key value pairs for search filters.
```

Comment:
Suggested rewording:

Metadata queries can also be performed using non-positional parameters with the `~astroquery.mast.MastMissionsClass.query_criteria` function.

Reply:
@ceb8 I updated the documentation in the latest commit as per your suggestion.